{
  "parameters": {
    "endpoint": "https://contoso.communications.azure.com",
    "api-version": "2021-11-15-preview",
    "callRequest": {
      "callLocator": {
        "groupCallId": "0c1610d4-454d-4ac5-a455-05ba5dc6c5fd",
        "kind": "groupCallLocator"
      },
      "source": {
        "communicationUser": {
          "id": "8:acs:b9614373-fd0b-480c-8fd2-cb58b70eab9f_acacdeb5-dba7-479b-ab45-5e76469d87b2"
        }
      },
      "subject": "Incident meeting",
      "callbackUri": "https://app.contoso.com/callback",
      "requestedMediaTypes": [
        "audio"
      ],
      "requestedCallEvents": [
        "participantsUpdated"
      ]
    }
  },
  "responses": {
    "202": {
      "body": {
        "callConnectionId": "b9614373-fd0b-480c-8fd2-cb58b70eab9f"
      }
    },
    "400": {
      "body": {
        "error": {
          "code": "400",
          "message": "The request contains invalid data."
        }
      }
    },
    "401": {
      "body": {
        "error": {
          "code": "401",
          "message": "Request is not authorized."
        }
      }
    },
    "403": {
      "body": {
        "error": {
          "code": "403",
          "message": "User is not allowed to perform specified action."
        }
      }
    },
    "404": {
      "body": {
        "error": {
          "code": "404",
          "message": "Resource not found on the server."
        }
      }
    },
    "500": {
      "body": {
        "error": {
          "code": "500",
          "message": "Internal server error."
        }
      }
    }
  }
}